lisa/tools/mkfs.py

```diff
@@ -2,17 +2,31 @@
 # Licensed under the MIT license.
 import re
 from enum import Enum
-from typing import cast
+from typing import Optional, Type, cast
 
 from lisa.executable import Tool
 from lisa.operating_system import Posix, Suse
-from lisa.util import LisaException, get_matched_str
+from lisa.util import LisaException, find_patterns_groups_in_lines, get_matched_str
 
 FileSystem = Enum(
     "mkfs",
-    ["xfs", "ext2", "ext3", "ext4", "btrfs", "hugetlbfs", "nfs", "tracefs"],
+    [
+        "xfs",
+        "ext2",
+        "ext3",
+        "ext4",
+        "btrfs",
+        "hugetlbfs",
+        "nfs",
+        "tracefs",
+        "ufs",
```

Comment:
I recall for freebsd, it also has zfs file system type?

Reply:
Fio tests work with the most common disk format for the distro, which for bsd is `ufs`. There are other file systems as well, but they are currently not required so I skipped adding them.